src/__tests__/element-queries.js

```diff
@@ -1182,3 +1182,65 @@ test('return a proper error message when no label is found and there is an aria-
 </div>"
 `)
 })
+
+// https://github.com/testing-library/dom-testing-library/issues/723
+it('gets form controls by label text on IE and other legacy browsers', () => {
+  // Simulate lack of support for HTMLInputElement.prototype.labels
+  const inputLabelsSpy = jest
+    .spyOn(HTMLInputElement.prototype, 'labels', 'get')
+    .mockReturnValue(undefined)
```

Comment:
This is a great idea!

Reply:
Thanks, but all credit goes to @eps1lon's [reference implementation](https://github.com/eps1lon/dom-accessibility-api/pull/352/files#diff-227df8dcfa07fb56cfb8e394fb09ada7R443) 🙂